zerver/tests/test_subs.py

```diff
@@ -1174,7 +1174,7 @@ def test_remove_invalid_user(self) -> None:
                                      "principals": ujson.dumps(["baduser@zulip.com"])})
         self.assert_json_error(
             result,
-            "User not authorized to execute queries on behalf of 'baduser@zulip.com'",
+            "Email 'baduser@zulip.com' not recognized",
```

Comment:
Can you split the backend changes as a separate commit?  I'm not sure they're actually coupled to the frontend changes.